### DISCUSSION TITLE
How to tell command is run in a zellij session?

### DISCUSSION THREAD

**Discussion by al3xandru** [zellij-org/zellij#1257] [General]
Is there an env variable or any other mechanism to determine that a command/program is run in a zellij session? (e.g. tmux is setting a `$TMUX` env variable)

**Comment by al3xandru** [comment]
Effectively 1second after posting, I thought to test if there isn't a `$ZELLIJ` env variable. Pleasant surprise!

**Comment by a-kenji** [comment]
Awesome! 
Here are a few simple examples of integration: https://zellij.dev/documentation/integration.html
You can also set specific environment variables per layout you are loading.

I personally use it to modify my shell prompt (starship) to show the prompt either in the prompt, or in the pane frame:
https://user-images.githubusercontent.com/65275785/132888646-628c66bc-42e5-4852-adb0-581e1ef157a9.gif

Ignore the prompt for now, that is a current pr.
